etcdctl/ctlv3/command/user_command.go

```diff
@@ -93,7 +93,7 @@ func userAddCommandFunc(cmd *cobra.Command, args []string) {
 	if !passwordInteractive {
 		fmt.Scanf("%s", &password)
 	} else {
-		password = readPasswordInteractive(args[0])
+		password = readPasswordInteractive(args[0], true)
```

Comment:
Oops, sorry for that. The flag is obsolete because `auth authenticate` command doesn't exist now.